index_de.md

```diff
@@ -4,52 +4,52 @@ layout: signed
+mit Beiträgen wie dem GNU Betriebssystem und Emacs.
+
+In jüngster Zeit wurden eine reihe verwerflicher Online-Angriffe
+bewerkstelligt, mit dem Ziel ihn aus dem Vorstand der FSF zu entfernen,
+weil er seine persönlichen Meinungen zum Ausdruck gebracht hat. Wir
+mussten schon öfter dabei zuschauen, wie dies bereits anderen Aktivisten
+der Freien Software-Bewegung passiert ist. Wir wollen nicht weiter
+tatenlos dabei zusehen, wie ein Symbol dieser Gemeinschaft angegriffen
+wird.
 
 Die FSF ist ein autonomes Gremium, welches durchaus in der Lage ist, 
-seine Mitglieder fair und unvoreingenommen zu behandeln und sollte 
-dem sozialen Druck von außen nicht nachgeben. Wir fordern die 
-FSF auf, die Argumente gegen RMS objektiv zu betrachten
-und die wahre Bedeutung seiner Worte und Taten zu verstehen.
-
-In der Vergangenheit hat RMS seine Ansichten in einer Weise 
-zum Ausdruck gebracht, die viele Menschen verärgerte. Er konzentriert 
-sich in der Regel mehr auf den philosophischen Hintergrund sowie 
-auf das Nacheifern nach Wahrheit und pflegt sprachlichen Purismus. 
-Auf der anderen Seite räumt er den Gefühlen der Menschen in seinen 
-Kommentaren oft zu wenig Raum ein. 
-
-Das macht seine Argumente anfällig für Missverständnisse und 
-Falschdarstellungen, was unserer Meinung nach aus dem offenen Brief, 
-in dem seine Absetzung gefordert wird, klar hervorgeht.
-Seine Worte müssen unter Berücksichtigung, dass er in den meisten Fällen 
-die Dinge nicht diplomatisch zum Ausdruck bringt, interpretiert werden. 
-
-Unabhängig davon: Stallman's Meinungen, für die er verfolgt wird, 
-sind nicht relevant hinsichtlich seiner Fähigkeit, eine Gemeinschaft wie 
-die FSF zu führen. 
-Darüber hinaus hat er genauso Anspruch auf seine eigenen Meinungen 
-wie jeder andere auch. Mitglieder und Unterstützer 
-müssen nicht zwingend mit seinen Ansichten übereinstimmen, 
-sollten aber sein Recht auf Gedanken- und Redefreiheit respektieren.
+seine Mitglieder fair und unvoreingenommen zu behandeln und sollte
+politischem Druck von außen nicht nachgeben. Wir fordern die FSF auf,
+die Argumente gegen RMS objektiv zu betrachten und die eigentliche
+Bedeutung seiner Worte und Taten — im lichte der persönlicher Freiheit,
+die ja den Hauptwert der Freien Software Bewegung darstellt — zu
+verstehen.
+
+In der Vergangenheit hat RMS seine Ansichten in einer Weise zum
+Ausdruck gebracht, die durchaus verärgerten könnte. Er konzentriert 
+sich in der Regel mehr auf philosophische Hintergründe sowie auf die Suche
+nach Wahrheit und pflegt sprachlichen Purismus. Dabei räumt er den
+Gefühlen der Menschen in seinen Kommentaren oft wenig Raum ein. 
+Das macht seine Argumente für Missverständnisse und Verstellungen
+anfällig, was unserer Meinung nach aus dem offenen Brief, in dem seine
+Absetzung gefordert wird, klar hervorgeht. Es muss in der Interprätation
+seiner Worte daher berücksichtigt werden, dass eine diplomatische
+Ausdrucksweise weder sein Metier ist, noch unabdingbar für seinen
```

Comment:
"Metier" = "Fachbereich".
So if he engages in "language purism", diplomacy could indeed be his "Metier". 
But, IMO the original text says more like "it is not his goal to ... pursue diplomacy".
Proposal: "seine undiplomatische Ausdrucksweise keine Voraussetzungs ist für seinen aktivistischen Erfolg"?